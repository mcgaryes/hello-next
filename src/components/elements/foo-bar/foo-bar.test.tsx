import {render} from '@testing-library/react'
import FooBar from "@/elements/foo-bar/foo-bar";

test('Renders foo-bar unchanged', () => {
    const {container} = render(<FooBar value={"foobar"}/>)
    expect(container).toMatchSnapshot()
})
