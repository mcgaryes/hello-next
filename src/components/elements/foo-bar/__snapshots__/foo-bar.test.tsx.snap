// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders foo-bar unchanged 1`] = `
<div>
  <div>
    foobar
  </div>
</div>
`;
